src/modules/fundingManager/bondingCurve/FM_BC_Bancor_Redeeming_VirtualSupply_v1.sol

```diff
@@ -598,4 +598,27 @@ contract FM_BC_Bancor_Redeeming_VirtualSupply_v1 is
             );
         }
     }
+
+    //--------------------------------------------------------------------------
+    // Overridden Internal Functions
+
+    /// @notice    Distributes issuance tokens by minting them to the receiver.
```

Comment:
Naming is still old in the Natspec, let's not talk about distributing.

Reply:
Done